src/backend.cpp

```diff
@@ -998,6 +1004,13 @@ void BackEnd::removeRecentConnection(const QString &name)
     }
 }
 
+void BackEnd::regenerateOwnCertificate()
+{
+    X509Certificate::createCertificate(defaultPkiPath());
+    mOwnCertificateItemModel->updateCertificateList();
+    emit ownCertificateItemModelChanged();
```

Comment:
The *model* doesn’t change, the property should be `CONSTANT`, and the model signal a model reset.

Reply:
Done.